docs/pages/installation/gitpod.md

```diff
@@ -0,0 +1,134 @@
+  - [Installation: Using `mesheryctl`](#installation-using-mesheryctl)
+  - [Installation: Using Helm](#installation-using-helm)
+  - [Installation: Manual Steps](#installation-manual-steps)
+  - [Access Meshery UI](#access-meshery-ui)
+
+# In-cluster Installation
+
+Follow the steps below to install Meshery in your Minikube cluster.
+
+## Preflight Checks
+
+Read through the following considerations prior to deploying Meshery on Minikube.
+
+### Preflight: Cluster Connectivity
+
+
+You can develop and run Meshery in a Gitpod using your choice of tool:
+
+- A command shell, via an SSH connection initiated using GitHub CLI.
+- One of the JetBrains IDEs, via the JetBrains Gateway.
+- The Visual Studio Code desktop application.
+- A browser-based version of Visual Studio Code.
+#### Fork and Open in Gitpod:
+- Open your forked Meshery repository on GitHub.
+- Click on the "Gitpod" button in the top right corner of the repository page (only visible with the <a href="https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki" class="meshery-light">Gitpod browser extension</a> installed).
+
+{% include alert.html type="dark" title="About Gitpod" content="Gitpod will automatically clone and open the repository for you in VSCode by default. It will also automatically build the project for you on opening and comes with Docker and other tools pre-installed making it one of the fastest ways to spin up an environment for <a href='https://github.com/meshery/meshery'>Meshery.</a>" %}
+
+Start the minikube, if not started using the following command:
+{% capture code_content %}minikube start --cpus 4 --memory 4096{% endcapture %}
+{% include code.html code=code_content %}
+Please allocate cpus based on the machine you selected in the Github codespaces and to check up on your minikube cluster :
+{% capture code_content %}minikube status{% endcapture %}
+{% include code.html code=code_content %}
+Verify your kubeconfig's current context.
+{% capture code_content %}kubectl cluster-info{% endcapture %}
+{% include code.html code=code_content %}
+
+## Installation: Using `mesheryctl`
+
+Use Meshery's CLI to streamline your connection to your Minikube cluster. Configure Meshery to connect to your Minikube cluster by executing:
+
+{% capture code_content %}$ mesheryctl system config minikube{% endcapture %}
+{% include code.html code=code_content %}
+
+Once configured, execute the following command to start Meshery.
+
+{% capture code_content %}$ mesheryctl system start{% endcapture %}
+{% include code.html code=code_content %}
+
+If you encounter any authentication issues, you can use `mesheryctl system login`. For more information, click [here](/guides/mesheryctl/authenticate-with-meshery-via-cli) to learn more.
+
+## Installation: Using Helm
+
+For detailed instructions on installing Meshery using Helm V3, please refer to the [Helm Installation](/installation/helm) guide.
```

Comment:
@MonalikaPatnaik! This should be /installation/kubernetes/helm; currently, it is showing a 404 error. Could you please update it for CodeSpaces as well? It's incorrect there too